ui/src/app/applications/components/applications-list/applications-filter.tsx

```diff
@@ -58,6 +58,15 @@ class ItemsFilter extends React.Component<{
     }
 }
 
+function getLabelsSuggestions(labels: Map<string, Set<string>>) {
```

Comment:
why function rather than const getLabelSuggestions(label:..) => {..}?

Reply:
we don't need to preserve `this` value, so I chose to use function